Userland/Libraries/LibMain/Main.cpp

```diff
@@ -24,12 +24,10 @@ int main(int argc, char** argv)
     });
     if (result.is_error()) {
         auto error = result.release_error();
-        if (error.is_syscall())
-            warnln("Runtime error: {}: {} (errno={})", error.string_literal(), strerror(error.code()), error.code());
-        else if (error.is_errno())
-            warnln("Runtime error: {} (errno={})", strerror(error.code()), error.code());
+        if (error.is_errno())
+            warnln("Runtime error: {}: {} (errno={})", error.where(), strerror(error.code()), error.code());
         else
-            warnln("Runtime error: {}", error.string_literal());
+            warnln("Runtime error: {}: {}", error.where(), error.string_literal());
```

Comment:
Is the removal of the `error.is_syscall()` branch intentional?
It will now no longer output `strerror(...)` in that case.

Reply:
`is_errno()` returns true for both syscalls and errno errors, so removed branch (output is the same for both cases).

Reply:
Ah right, I guess it only changes if there are errno Errors which are not syscalls but I don't know if those exist.
Seems fine then :+1: